src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -103,18 +104,20 @@ class InstallAppsTask extends AbstractServerTask {
 
     private void installMultipleApps(List<Task> applications, String appsDir) {
         applications.each{ Task task ->
-          installProject(task, appsDir)
+            installProject(task, appsDir)
         }
     }
 
     private void installProjectArchive(Task task, String appsDir) {
-        if (task.name == 'bootJar' || task.name == 'bootRepackage') {
+        if("springboot".equals(getPackagingType())) {
```

Comment:
Is it ok that the passed in Task is ignored here? We used to key off of that.

Reply:
The correct task in case of spring boot war case was not getting passed here. So I completely removed the dependency on passed task. Instead, I use task found by findSpringBootTask() method. But the `task` is used in else part.